cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -51,6 +51,7 @@ type applyFlags struct {
 	force              bool
 	dryRun             bool
 	etcdUpgrade        bool
+	externalEtcd       bool
```

Comment:
I don't see why we need a var.  We only support static manifest so triggering on that seems sufficient.

Reply:
Accidental cruft left from rebasing and cleaning this up after sitting for a while, will remove.